screens/StudentLife.jsx

```diff
@@ -11,7 +14,10 @@ function StudentLife({ route }) {
     };
     return (
         <Main name={name} coverImage={coverImage}>
-            <View style={styles.center}>{/* No Content */}</View>
+            <View style={styles.center}>     
```

Comment:
Some of the links are not visible at the bottom. Let's make the entire content scrollable instead of making the links scrollable.

Reply:
> Some of the links are not visible at the bottom. Let's make the entire content scrollable instead of making the links scrollable.

You can change the View to ScrollView on line 17 to make that happen.